toolbox/deps_update/githubClient.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2017 Istio Authors
```

Comment:
I think it might be a go thing to name files snake case `github_client.go`

but not a  big deal i don't think.